frontend/assignments/3/frontend/src/components/dashboard/Dashboard.tsx

```
@@ -0,0 +1,80 @@
+
+  let content;
+  if (isLoading) {
+    content = <LinearProgress />;
+  } else if (isError) {
+    content = <div>Error fetching data</div>;
+  } else {
+    content = (
+      <>
+        <FlexDivContainer width="10em">
+          <Underline active={exploreTabActive}>
+            <div onClick={() => handleTabClick('explore')} style={exploreTabStyle}>
```

Biome (1.9.4)

[error] Enforce to have the onClick mouse event with the onKeyUp, the onKeyDown, or the onKeyPress keyboard event.

Actions triggered using mouse events should have corresponding keyboard events to account for keyboard-only navigation.

(lint/a11y/useKeyWithClickEvents)